target/linux/lantiq/files/arch/mips/boot/dts/FRITZ7362SL.dts

```diff
@@ -68,7 +52,7 @@
 &spi {
 	status = "okay";
 	pinctrl-names = "default";
-	pinctrl-0 = <&pins_spi_default>;
+	pinctrl-0 = <&spi_pins>, <&spi_cs4_pins>;
```

Comment:
I would move this to the SoC node, The CS4 is probably also used in the bootrom and there it is probably only possible to use CS4 and nothing else for SPI NOR. 
If someone uses a different CS we can overwrite the SoC setting

Reply:
I didn't think that this may have been "hardcoded" in bootrom, but looking at all boards and that they're all using CS4 it may indeed be.

the latest update moves these pinctrl-settings to soc.dtsi and drops the Danube SPI changes because all Danube boards either use NOR or NAND (none uses SPI).